Gravity-tube/src/Pages/SingleVideoPage/Component/VideoDetailPage.js

```diff
@@ -0,0 +1,153 @@
+import React, { useEffect, useState } from "react";
+import { AiOutlineDislike, AiOutlineLike } from "react-icons/ai";
+import { MdOutlineWatchLater } from "react-icons/md";
+import { RiPlayListAddFill } from "react-icons/ri";
+import img from "../../../Assets/Images/gt-logo.png";
+import Avatar from "../../../Components/Avatar/Avatar";
+import VideoCard from "../../../Components/VideoCard/VideoCard";
+import ReactPlayer from 'react-player/youtube'
+import "./VideoDetailPage.css";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+import { useVideoData } from "../../../Contexts/Videos-context";
+
```

Comment:
this file should be in the component folder don't make components folder inside a page folder